src/components/cabinList/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import cabinData from '../../data/cabin';
+import '../../App.css';
+
```

Comment:
also - move the css file import into App.  You just need to import it once at the top level